content/docs/tasks/security/app-health-check.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Health Check on Istio Services
+description: Shows how to do health check for Istio services.
+weight: 65
+keywords: [security,health-check]
+---
+
+This task shows how to use [Kubernetes liveness and readiness probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) for health check on Istio services.
+
+There are three options for liveness and readiness probe in Kubernetes: 1) command; 2) http request; 3) tcp request. In this task, we provide examples for the first two options with Istio mutual TLS enabeld and disabled, respecitively. We will cover the third option soon.
+
+## Before you begin
+
+* Understand [Kubernetes liveness and readiness probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/), Istio [authentication policy](/docs/concepts/security/authn-policy/) and [mutual TLS authentication](/docs/concepts/security/mutual-tls/) concepts.
+
+* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (e.g use `install/kubernetes/istio-demo.yaml` as described in [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps), or set `global.mtls.enabled` to false using [Helm](/docs/setup/kubernetes/helm-install/)).
```

Comment:
Make a link to the YAML. The link should use the special marker {{< branch_name >}} to refer to the current branch in the GitHub link.

Reply:
Done.